sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/datetimeExpressions.scala

```diff
@@ -3011,13 +3011,17 @@ object SubtractDates {
   arguments = """
     Arguments:
       * sourceTz - the time zone for the input timestamp
+          - if not specified, SESSION_LOCAL_TIMEZONE is used when sourceTs is timestamp_ntz,
```

Comment:
`SESSION_LOCAL_TIMEZONE` is not valid for end users. Point out the SQL config `spark.sql.session.timeZone` or better `SESSION_LOCAL_TIMEZONE.key`